var/spack/repos/builtin/packages/openkim-models-v2/package.py

```diff
@@ -0,0 +1,33 @@
+# Copyright 2013-2019 Lawrence Livermore National Security, LLC and other
+# Spack Project Developers. See the top-level COPYRIGHT file for details.
+#
+# SPDX-License-Identifier: (Apache-2.0 OR MIT)
+
+from spack import *
+
+
+class OpenkimModelsV2(CMakePackage):
+    """OpenKIM is an online framework for making molecular simulations
+       reliable, reproducible, and portable. Computer implementations of
+       inter-atomic models are archived in OpenKIM, verified for coding
+       integrity, and tested by computing their predictions for a variety
+       of material properties. Models conforming to the KIM application
+       programming interface (API) work seamlessly with major simulation
+       codes that have adopted the KIM API standard.
+    """
+    homepage = "https://openkim.org/"
+    url      = "https://s3.openkim.org/archives/collection/OpenKIM-Models-v2-2019-02-21.txz"
+    git      = "https://github.com/openkim/kim-api.git"
```

Comment:
This is the same git URL as for `kim-api-v2`: do the `cmake_args` restrict what is installed for the `devel` version? If not, I get the impression this might be installing the same thing twice. That's fine if that's what is intended, although it would likely generate conflicts when using `spack view` for example.

Reply:
Ah, this was a copy and paste that didn't get updated.  I'll fix up this and the docstring to be more appropriate for the openkim-models-v2 package.